modules/deploy/partials/kubernetes/guides/create-storageclass.adoc

```diff
@@ -24,13 +25,22 @@ ifndef::env-eks[]
 helm repo add metal-stack https://helm.metal-stack.io
 helm repo update
 helm install csi-driver-lvm metal-stack/csi-driver-lvm \
+  --version 0.6.0 \
```

Comment:
Right now the highlighted line in this code block is `--create-namespace  \`, just want to double check if we wanted to keep it there.

Reply:
Good catch - thanks!